Extensions/blacklist.js

```diff
@@ -561,8 +567,8 @@ XKit.extensions.blacklist = new Object({
 					m_content = $(this).find(".caption").html();
 				}
 
-				if ($(this).find(".reblog-list-item").length > 0) {
-					m_content = $(this).find(".reblog-list-item").map(function() {
+				if ($(this).find(".reblog-list-item, .reblog-content").length > 0) {
```

Comment:
IIRC `.reblog-content` is a selector used on desktop too, so we should scope it to mobile-only using something like `.mh-post .reblog-content`, so we don't change existing desktop behavior.